app/graphql/resolvers/queries/get-metadata.js

```diff
@@ -0,0 +1,36 @@
+import db from '../../../data/index.js'
```

Comment:
Should we organise the `graphql` folder differently to have more consistency. 

We could match it to the format of other folders eg by eventtype so it could look like this

```
graphql
  resolvers
    index.js
    comms-message
      index.js
      get-comms-by....
    file-metadata
      index.js
      get-metadata
```

That way we can import the index for each folder into the resolvers index and that can be put directly into the apollo server.